PWGJE/EMCALJetTasks/TrackEfficiencyConfiguration.yaml

```diff
@@ -0,0 +1,14 @@
+
+nPtBins: 16
+ptBinning: [0.0, 0.5, 1.0, 2.0, 3.0, 4.0, 5.0, 6.0, 7.0, 8.0, 9.0, 10.0, 11.0, 12.0, 13.0, 14.0, 100.0]
+
+protonProton:
+    LHC17p: []
+
+leadLead:
+    LHC18q: 
+        central: [0.964, 0.964 , 0.939 , 0.936 , 0.949 , 0.958 , 0.976 , 0.967 , 0.967 , 0.967 , 0.967 , 0.976 , 0.976 , 0.984 , 0.976 , 0.976]
```

Comment:
done. I also removed the lead-lead heading, I don't think it is needed since the period (i.e. LHC18q) corresponds also to the collision system, and it is obvious to the user which system they are running over